aggregator/src/aggregator/http_handlers/tests/report.rs

```diff
@@ -389,6 +400,28 @@ async fn upload_handler() {
         &test_conn,
         "access-control-allow-origin" => "https://example.com/"
     );
+
+    // Reports that aren't aligned to the task time precision must be rejected
+    clock.advance(&Duration::from_seconds(1));
+    let bad_leader_time_alignment_report = create_report(
+        &leader_task,
+        &hpke_keypair,
+        clock.now(), /* not aligned! */
```

Comment:
super annoying nit: evidently this test passes so this is fine... but I feel like it'd be cleaner to do something like `clock.now().add(&Duration::from_seconds(1));`, because what if the `clock` is shared via `Arc<>` into somewhere else, and advancing it here messes with some other state?